com.hudren.woodpile/src/com/hudren/woodpile/net/SerializedLayoutReader.java

```diff
@@ -60,6 +61,9 @@ else if ( someEvent instanceof org.apache.logging.log4j.core.LogEvent )
 					else if ( someEvent instanceof LogEvent )
 						event = (LogEvent) someEvent;
 
+					else if ( someEvent instanceof ILoggingEvent )
+						event = new LogEvent( host, (ILoggingEvent) someEvent);
```

Comment:
I would move this check before the 'instances LogEvent' check for performance reasons